lib/ets/acceptor.ex

```diff
@@ -0,0 +1,40 @@
+defmodule ETS.Acceptor do
+  @moduledoc """
+  For processes which may receive ownership of an ETS table unexpectedly, the module
+  should include at least one acceptor.  Acceptors are defined by first importing the
+  `ETS.Acceptor` module, then calling the `accept` macro.  For example:
+  ```
+  defmodule Receiver do
+    use GenServer
+    import ETS.Acceptor
+
+    accept Set, set, _from, _gift, state do
+      new_state = Map.put(state, :set, set)
+      {:noreply, new_state}
+    end
+  ```
+  See `accept/6` for more details.
+  """
```

Comment:
There are so many ways to implement this; I chose to make a new module instead of putting it into `ETS.Utils` for a clean import statement that doesn't bring along any other unneeded macros.  I'm not certain about the "acceptor" concept.  Maybe it would be better to just call the module `ETS.Accept` and phrase it as "the module has an accept clause"?